grpc-gateway/service/grpcApi.go

```diff
@@ -43,23 +46,45 @@ func Register(server *grpc.Server, handler *RequestHandler) {
 	pb.RegisterGrpcGatewayServer(server, handler)
 }
 
-func NewRequestHandlerFromConfig(config HandlerConfig, clientTLS *tls.Config) (*RequestHandler, error) {
-	svc := config.Service
+func NewRequestHandlerFromConfig(config Config, clientTLS *tls.Config) (*RequestHandler, error) {
 	rdConn, err := grpc.Dial(
-		svc.ResourceDirectoryAddr,
+		config.ResourceDirectoryAddr,
 		grpc.WithTransportCredentials(credentials.NewTLS(clientTLS)),
 	)
 	if err != nil {
 		return nil, fmt.Errorf("cannot connect to resource aggregate: %w", err)
 	}
 	resourceDirectoryClient := pb.NewGrpcGatewayClient(rdConn)
 
+	raConn, err := grpc.Dial(
+		config.ResourceAggregateAddr,
+		grpc.WithTransportCredentials(credentials.NewTLS(clientTLS)),
+	)
+	if err != nil {
+		return nil, fmt.Errorf("cannot connect to resource aggregate: %w", err)
+	}
+	resourceAggregateClient := raService.NewResourceAggregateClient(raConn)
+
+	pool, err := ants.NewPool(config.GoRoutinePoolSize)
+	if err != nil {
+		return nil, fmt.Errorf("cannot create goroutine pool: %w", err)
+	}
+
+	resourceSubscriber, err := nats.NewSubscriber(config.Nats, pool.Submit, func(err error) { log.Errorf("grpc-gateway: error occurs during receiving event: %v", err) }, nats.WithTLS(clientTLS))
+	if err != nil {
+		return nil, fmt.Errorf("cannot create resource nats subscriber %w", err)
```

Comment:
change formulation; proposal was already mentioned in previous comment.

Reply:
fixed